RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -23,6 +23,9 @@
 from RecoJets.JetProducers.fixedGridRhoProducerFastjet_cfi import *
 fixedGridRhoFastjetAllTmp = fixedGridRhoFastjetAll.clone(pfCandidatesTag = cms.InputTag("particleFlowTmp"))
 
+from RecoJets.JetProducers.ak4PFJets_cfi import *
+ak4PFJetsTmp = ak4PFJets.clone(src = cms.InputTag("particleFlowTmp"))
```

Comment:
I can at least move it to the toModify so that it only runs for phase 2.

Beyond that the issue is that we need jets before the final PFCandidates are produced.  The only way I could see to avoid a duplication of the clustering is to add a new producer specifically for the final ak4PFJets to re-link the references of the ak4PFJetsTmp instead of reclustering.